src/imports/__tests__/otp.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @jest-environment node
+ */
+
+import jwt from 'jsonwebtoken'
+import MockAdapter from 'axios-mock-adapter'
+
+import OTP from '../otp'
+import config from '../../server/server.config'
+
+describe('OTP', () => {
+  it('should generate a valid JWT token', () => {
```

Comment:
maybe also a test to check that JWT is actually used in the request the server sends